### PR TITLE
Develop 3

### PR DESCRIPTION
**Pull request by LilJohny**

### CODE REVIEW

**Comment by LilJohny** [comment]
json-server needs to be launched separately with `npx json-server --watch api/db.json --port 3001`

File: src/components/NavBar.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ class NavBar extends React.Component {
             button = <BackToTopButton anchor={this.toolbar} />;
         }
         const { classes } = this.props;
-        const navbar_links = [["Home", "/"], ["Categories", "/categories"], ["Interesting Posts For You", "/interesting"], ["Your Inbox", "/inbox"]];
+        const navbar_links = [["Home", "/"], ["Categories", "/categories"], ["Interesting Posts For You", "/interesting"], ["Your Inbox", "/inbox"], ["Register account","/registration"]];
```
Comment (pavlo-berezin) [inline src/components/NavBar.js:59]: please use camel case

File: src/components/RegistrationForm.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        this.state = { name: '', surname: '', mail: '', password: '' };
+
+        this.handleChange = this.handleNameChange.bind(this);
+        this.handleSurnameChange = this.handleSurnameChange.bind(this);
+        this.handleMailChange = this.handleMailChange.bind(this);
+        this.handlePasswordChange = this.handlePasswordChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleNameChange(event) {
```
Comment (pavlo-berezin) [inline src/components/RegistrationForm.js:38]: you can handle all this changes with one change handler.

File: src/components/RegistrationForm.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    handleSubmit(event) {
+        alert('A name was submitted: ' + this.state.value);
+        event.preventDefault();
+    }
+    render() {
+        const { classes } = this.props;
+        return (
+            <Paper className={classes.root}>
+                <form onSubmit={this.handleSubmit}>
+                    <TextField className={classes.field} required id="standard-required" label="Your Name" onChange={this.handleNameChange} /><br />
```
Comment (pavlo-berezin) [inline src/components/RegistrationForm.js:60]: id should be unique

**Pull request merged**